src/components/document/component/DeleteDocumentModal.tsx

```diff
@@ -0,0 +1,50 @@
+import {Modal, ModalBody, ModalFooter, ModalHeader} from "baseui/modal";
+import {Paragraph2} from "baseui/typography";
+import {Block} from "baseui/block";
+import {Button} from "baseui/button";
+import {intl} from "../../../util";
+import * as React from "react";
+
+type ModalDeleteProps = {
+  title: string,
+  isOpen: boolean,
+  documentName: String,
+  submit: Function,
```

Comment:
Ikke bruk `Function`

Reply:
Jeg tror begge to er bra å bruke.

Reply:
Function sier ikke noe om funksjonen, det er som å skrive Object istedenfor "InformationType", så fix pls